public/admin/config.yml

```diff
@@ -27,6 +27,7 @@ collections:
     fields:
       - { label: "Title", name: "title", widget: "string" }
       - { label: "Date", name: "date", widget: "datetime", hint: "Publish date." }
+      - { label: "Type", name: "type", widget: "select", options: ["General", "Product"], hint: "Type of FAQ" }
```

Comment:
im assuming the cms admin has the options of adding a type for the faq?

Reply:
if u mean the person who is updating or adding the cms contents will be able to select the type of FAQ, then yes, this will allow the admin to select the type of FAQ to update or add onto the website.

Reply:
👍